workflow-dev/tpl/deis-workflow-manager-rc.yaml

```diff
@@ -25,19 +25,19 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: WORKFLOW_MANAGER_PORT
+        - name: PORT
           value: "8080"
-        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+        - name: VERSIONS_API_URL
           value: {{.workflowManager.versionsApiURL}}
-        - name: WORKFLOW_MANAGER_DOCTORAPIURL
+        - name: DOCTOR_API_URL
```

Comment:
good idea. fixed in https://github.com/deis/charts/pull/268/commits/204671dd6f49f3c42daa3c1758b83404bee94010